### PR TITLE
New interview on firefox

### PR DESCRIPTION
**Pull request by Michael-Salato-DU**

### CODE REVIEW

File: NGTrackForce/src/app/components/myinterview-view/myinterview-view.component.html [hunk 1]
```diff
@@ -85,10 +95,23 @@ <h3>My Interviews</h3>
...
             <td>
               <form [formGroup]="registerForm" >
                   <div class="input-group mb-3" style="width: 310px;">
-                      <input class="form-control" type="datetime-local" formControlName="dateinput">
+
+                      <!-- <label for="updateDate">Interview Date:</label> -->
+                      <input type="date"
+                        class="form-control" name="updateDate" id="updateDate" formControlName="updateDate"
+                        placeholder="mm/dd/yyyy" min = "{{today | date:'yyyy-MM-dd'}}">
+                      <!-- <label for="updateTime">Interview Time:</label> -->
```
Comment (ryan-schlientz) [inline NGTrackForce/src/app/components/myinterview-view/myinterview-view.component.html]: Place remove this commented code as well as the one a few lines up.

File: NGTrackForce/src/app/components/myinterview-view/myinterview-view.component.ts [hunk 2]
```diff
@@ -236,33 +250,15 @@ export class MyInterviewComponent implements OnInit {
...
-  // ============================================
-  saveInterview(interview: Interview) {}
-
-
-  // THIS METHOD IS REPLACED BY STORING THE CLIENTS IN LOCAL STORAGE
-  // getClientNames() {
-  //   this.clientService.getAllClients().subscribe(data => {
-  //     this.clients = data;
-  //   });
-  // }
```
Comment (ryan-schlientz) [inline NGTrackForce/src/app/components/myinterview-view/myinterview-view.component.ts]: I this code still needed for the future? If not, remove it.

File: TrackForce/src/test/java/com/revature/test/TestNG/AssociateTests.java [hunk 3]
```diff
@@ -103,25 +103,26 @@ public void addInterview() {
...
 		date.sendKeys(Keys.ARROW_UP);
 		
 		AssociateHome.checkBox(wd).click();
-		wd.manage().timeouts().implicitlyWait(20, TimeUnit.SECONDS);
+		//wd.manage().timeouts().implicitlyWait(20, TimeUnit.SECONDS);
 		AssociateHome.addInterview(wd).click();
 		
 		
-		wd.manage().timeouts().implicitlyWait(20, TimeUnit.SECONDS);
+		//wd.manage().timeouts().implicitlyWait(20, TimeUnit.SECONDS);
```
Comment (ryan-schlientz) [inline TrackForce/src/test/java/com/revature/test/TestNG/AssociateTests.java]: Remove all commented lines of code from this file.

File: NGTrackForce/src/app/components/myinterview-view/myinterview-view.component.html [hunk 4]
```diff
@@ -85,10 +95,23 @@ <h3>My Interviews</h3>
...
 
+            <!-- 
+            Interview Update 
+            -->
             <td>
               <form [formGroup]="registerForm" >
                   <div class="input-group mb-3" style="width: 310px;">
-                      <input class="form-control" type="datetime-local" formControlName="dateinput">
+
+                      <!-- <label for="updateDate">Interview Date:</label> -->
```
Comment (asaadpeter) [inline NGTrackForce/src/app/components/myinterview-view/myinterview-view.component.html]: Is this an important comment code?

File: NGTrackForce/src/app/components/myinterview-view/myinterview-view.component.html (hunk 1 above)
Comment (asaadpeter) [inline NGTrackForce/src/app/components/myinterview-view/myinterview-view.component.html]: Is this an important comment code?

**Pull request merged**